tests/loads_from_flag/arguments.txt

```diff
@@ -0,0 +1,2 @@
+-mode=diff
+-custom-load=@some//repo:rules.bzl,custom_rule
```

Comment:
Is it possible to define custom directives that achieve the same effect?

Reply:
It is not, though I would like it to be - see https://github.com/bazelbuild/bazel-gazelle/issues/1313